server/configuration.go

```diff
@@ -181,6 +181,12 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(loadConfigErr, "failed to load plugin configuration")
 	}
 
+	// Setup translations
+	config := p.API.GetConfig()
+	if locError := p.Helpers.SetLocalization(&config.LocalizationSettings, "plugins/"+manifest.Id+"/assets"); err != nil {
```

Comment:
0/5, but maybe we should put the translation files in `assets/i18n` to keep them consistent with the other repos?

Reply:
Maybe even `assets/i18n/server`?

Reply:
Agree. Will change. I don't think `/server` is necessary though because the webapp translations aren't in assets (they get bundled with webpack).